src/extension/background-script/connectors/lnbits.ts

```diff
@@ -64,13 +66,69 @@ class LnBits implements Connector {
     return new Error("Not yet supported with the currently used account.");
   }
 
-  // not yet implemented
-  getInvoices() {
-    console.error(
-      `Not yet supported with the currently used account: ${this.constructor.name}`
-    );
-    return new Error(
-      `${this.constructor.name}: "getInvoices" is not yet supported. Contact us if you need it.`
+  /*
+  LNBits Swagger Docs: https://legend.lnbits.org/devs/swagger.html#/default/api_payments_api_v1_payments_get
+  Sample Response:
+    [
+      {
+        "checking_id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
```

Comment:
@bumi you know if `custom_records` are hidden in here somewhere?

Reply:
in the checking id? 
no, I guess that's their internal id.

where are the docs for this call?

Reply:
https://legend.lnbits.org/devs/swagger.html#/default/api_payments_api_v1_payments_get

Reply:
it does not describe the response data, does it?

Reply:
Maybe you need to authorize first?:
<img width="250" src="https://user-images.githubusercontent.com/1016218/192423870-c635e2df-e7f1-4c07-9df6-e897885bd5ee.png" />

On "try out":
<img width="250" src="https://user-images.githubusercontent.com/1016218/192423931-ab0194f0-5635-440d-a153-b7f24ac1dbcf.png" />

This shows some info:
https://legend.lnbits.org/devs/swagger.html#/default/api_payments_create_api_v1_payments_post
<img width="250" src="https://user-images.githubusercontent.com/1016218/192424020-fa6524d8-1143-4e2a-9ceb-f843f4fbed8a.png" />

Reply:
Here's the actual response:
![Screenshot from 2022-09-30 18-24-26](https://user-images.githubusercontent.com/64399555/193274165-982bbcd3-ef28-4683-8b9b-f406b4447bb7.png)

Reply:
@bumi looks liek no `custom_records` available?  
Should we merge this in first to provide general support first?